src/Database/LSMTree/Internal/MergeSchedule.hs

```diff
@@ -809,17 +809,11 @@ supplyCredits conf c levels =
 -- on the merging run.
 --
 -- Initially, 1 update supplies 1 credit. However, since merging runs have
--- different numbers of input runs/entries, we may have to a more or less
+-- different numbers of input runs\/entries, we may have to a more or less
 -- merging work than 1 merge step for each credit.
-scaleCreditsForMerge :: MergePolicyForLevel -> Ref (MergingRun m h) -> Credits -> MR.Credits
--- A single run is a trivially completed merge, so it requires no credits.
-scaleCreditsForMerge LevelTiering _ (Credits c) =
-    -- A tiering merge has 5 runs at most (one could be held back to merged
-    -- again) and must be completed before the level is full (once 4 more
-    -- runs come in).
-    MR.Credits (c * (1 + 4))
-
-scaleCreditsForMerge LevelLevelling mr (Credits c) =
+scaleCreditsForMerge :: MergePolicyForLevel -> Ref (MergingRun m h)
+                     -> Credits -> MR.Credits
```

Comment:
In the remainder of the module, line breaks in type signatures look different.

Reply:
done